code/modules/gear_presets/uscm_medical.dm

```diff
@@ -34,6 +34,7 @@
 	dress_hat = list()
 	dress_gloves = list(/obj/item/clothing/gloves/marine/dress)
 	dress_shoes = list(/obj/item/clothing/shoes/laceup)
+	dress_extra = list(/obj/item/storage/large_holster/ceremonial_sword/full, /obj/item/clothing/suit/storage/jacket/marine/dress/bridge_coat_grey)
```

Comment:
why are you also giving heads of department ceremonial swords?

Reply:
Whoops. I just coded the XO and Heads first. forgot to not copy the XO's sword

Reply:
should be fixed now tho